streams/src/test/java/org/apache/kafka/streams/state/internals/RocksDBSegmentedBytesStoreTest.java

```diff
@@ -414,8 +417,11 @@ public void shouldLogAndMeasureExpiredRecords() {
         LogCaptureAppender.setClassLoggerToDebug(RocksDBSegmentedBytesStore.class);
         final LogCaptureAppender appender = LogCaptureAppender.createAndRegister();
 
-        context.setStreamTime(Math.max(retention, segmentInterval) * 2);
```

Comment:
I can't map this to the `nextSegmentWindow` bounds above. Can you elaborate?

Reply:
The new code is a tighter bound. Previously we used `2 * max(retention, interval)`, but all we really need is `retention + interval`. Not sure why I did the previous code, it's guaranteed to be at least `retention + interval`, but it's pretty esoteric...